data/categories.txt

```diff
@@ -346,7 +346,7 @@ cs:2Bar|pivo|pití|jídlo|voda|pitná voda|tekutiny
 sk:2Bar|pivo|pitie|jedlo|voda|pitná voda|tekutiny
 nl:2Bar|2pub|2café|bier|drinken|eten
 zh-Hant:1酒吧|酒館|飲酒|PUB|吧台|飲食
-pl:2Bar|2pub|piwo|napoje|jedzenie
+pl:2Bar|alkohole|drinki|napoje|jedzenie
```

Comment:
To be precise typing "piwo" should be enough to find places with beer. But I won't stick with it - I'll put back pub as synonym of bar and opposite.